files/images/teamspeak_viewer/badges/paris_gamesweek_2014_details.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><title>paris-gamesweek-2014-details-badge</title><g id="Layer_1" data-name="Layer 1"><path d="M7.52,6.11A53.85,53.85,0,0,1,32,0,54.59,54.59,0,0,1,56.47,6.11l.93.46v31l-.06.43a41.44,41.44,0,0,1-9.93,17.86C42.43,61.06,36.65,64,32,64s-10.19-3.39-15.08-8.5A42.23,42.23,0,0,1,6.66,38l-.06-.43v-31l.92-.46ZM32,3.33A50.18,50.18,0,0,0,9.95,8.64V37.36a38.92,38.92,0,0,0,9.37,15.83c4.3,4.49,9.09,7.47,12.68,7.47,3.79,0,8.66-2.57,13-7.1a38,38,0,0,0,9.05-16.19V8.65A50.87,50.87,0,0,0,32,3.33Z" style="fill:#8190aa;fill-rule:evenodd"/><path d="M33.67,11.48v.14q0,.86.05,1.74t.09,1.74c.15,2.44.4,4.84.78,7.21a58.25,58.25,0,0,0,1.55,7,50,50,0,0,0,2.08,5.84h0a47,47,0,0,0,2.86,5.61c.11.16.22.32.32.49h0q.14.23.29.5l.23.36h0l.24.38h0q.49.74,1,1.5t1.1,1.48l.59.76.49.63H41.22l-.12-.16Q40.55,46,40,45.16c-.5-.72-1.27-2-1.77-2.68a6.81,6.81,0,0,0-.88-1,7.58,7.58,0,0,0-2.44-1.46,8.45,8.45,0,0,0-5.79,0,7.57,7.57,0,0,0-2.44,1.46,6.9,6.9,0,0,0-.6.61c-.71.82-1.44,2.16-2,3q-.55.79-1.1,1.53l-.12.16H18.6l.51-.64.61-.76h0q.6-.77,1.16-1.56T22,42.3h0l.17-.26h0l.16-.25c.11-.2.21-.39.32-.56s.23-.36.36-.54a47.09,47.09,0,0,0,2.85-5.59A50.09,50.09,0,0,0,27.9,29.3a57.48,57.48,0,0,0,1.39-6c.36-2,.62-4.1.8-6.19q.12-1.38.19-2.72t.1-2.74v-.55h0v0h.39l1.25,0,1.24,0h.39v.05h0v.4ZM31,30.37h2l0-.16q-.41-1.41-.74-2.8-.11-.47-.21-.94-.1.47-.21.94-.33,1.39-.74,2.8l0,.16Zm2.87,2.76H30.14q-.36,1-.77,2.06-.3.75-.62,1.48a11.53,11.53,0,0,1,1.16-.27,11.76,11.76,0,0,1,5.38.27q-.32-.74-.62-1.48Q34.26,34.14,33.9,33.13Z" style="fill:#8190aa;fill-rule:evenodd"/><path d="M19.12,25.55H18v2.9c-2.15-2-3.23-2.87-4.46-2.87A2.21,2.21,0,0,0,11.29,28a2.91,2.91,0,0,0,1.25,2.45l.8-.79A1.91,1.91,0,0,1,12.43,28a1.08,1.08,0,0,1,1.12-1.13c1,0,1.76.74,4.53,3.43h1Z" style="fill:#8190aa;fill-rule:evenodd"/><path d="M19.26,21.91c0-1.88-2.06-2.61-4-2.61s-4,.74-4,2.61,2,2.6,4,2.6,4-.73,4-2.6Zm-1.13,0c0,1-1.35,1.3-2.86,1.3s-2.84-.28-2.84-1.3,1.33-1.31,2.84-1.31S18.13,20.88,18.13,21.91Z" style="fill:#8190aa;fill-rule:evenodd"/><polygon points="19.12 15.53 11.41 15.53 11.41 16.65 13.52 18.73 14.3 17.98 13.06 16.81 19.12 16.81 19.12 15.53" style="fill:#8190aa;fill-rule:evenodd"/><path d="M19.12,10.15H17.34V9.34H16.21v.81h-4.8v1.63l4.83,2.75h1.1V11.42h1.78V10.15Zm-2.91,1.27v2l-3.46-2Z" style="fill:#8190aa;fill-rule:evenodd"/></g><g id="Ebene_1" data-name="Ebene 1"><rect width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect x="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect y="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect x="63.83" y="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/></g></svg>
```

Comment:
Why are there no line breaks here?

Reply:
Rechecked with original source. The file is untouched from original.